cpp/src/stream_compaction/copy_if.cuh

```diff
@@ -353,10 +353,9 @@ gdf_column copy_if(gdf_column const &input, Filter filter,
                           scatter_functor<Filter, block_size, per_thread>{},
                           output, input, block_offsets, filter,
                           input.valid != nullptr, stream);
-
     CHECK_STREAM(stream);
   }
-  
+
```

Comment:
Just a comment - this is a no-op file modification that is orthogonal to the work you're actually doing. Obviously the change is "safe", but it clutters up the changelog filelist for this commit and could be substantially confusing downstream.

Reply:
Agreed. I'll drop this change in a follow-up commit.

Reply:
Done